config/config.go

```diff
@@ -50,6 +50,7 @@ type UserPreferences struct {
 	DisableHTML          bool `yaml:"disable_html"`
 	DisableDownloads     bool `yaml:"disable_downloads"`
 	DisableNotifications bool `yaml:"disable_notifications"`
+	DisableShowurls      bool `yaml:"disable_show_urls"`
```

Comment:
```suggestion
	DisableShowURLs      bool `yaml:"disable_show_urls"`
```

(references need to be changed too)